missing dependencies

Moment and other dependencies are declared as peerdepency but nor in normal dependencies: 

```
Module not found: Error: Cannot resolve module 'moment' in .../node_modules/@gandi/react-translate/dist/tools
resolve module moment in .../node_modules/@gandi/react-translate/dist/tools
...
@ ./~/@gandi/react-translate/dist/tools/createTranslator.js 15:14-31
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,7 +4,11 @@
 
 ### Added
 
-  - Create a launcher for extract_messages, update_catalog and other scripts
+  - Create a launcher for extract_messages, update_catalog and other scripts.
+
+### Changed
+
+  - Clean dependencies and improve installation part of the README.
 
 ## [1.0.0-alpha] - 2016-11-17
 

--- a/README.md
+++ b/README.md
@@ -22,7 +22,7 @@
 npm install --save @gandi/react-translate
 ```
 
-[Optional] Add the following script to your `package.json`, it's a simple shortcut for extraction
+Add the following script to your `package.json`, it's a simple shortcut for extraction
 scripts:
 
 ```
@@ -32,6 +32,16 @@ scripts:
   "i18n": "react-translate-scripts"
 }
 ```
+
+Make sure you have `babel-cli` in your dependencies (it's up to you to select the version saved in
+your project):
+
+```
+npm install babel-cli
+```
+
+> Note for npm 2 users, you have to make sure po2json in the root `node_modules`, the same way as
+`babel-cli`
 
 ## Usage
 
@@ -281,11 +291,9 @@ Else you can start with [CONTRIBUTING.md](CONTRIBUTING.md).
 * Document `Pricing` features
 * Add formatCurrency component
 * Extract babeljs compilation process from extract_messages script (CRA compatible)
-* Add a script launcher and remove usage of `node_modules/.bin/bla-thing` from docs (cf. CRA)
 * Improve doc on how to load translations (at least give an example)
 * Link translator options to related libs
 * Remove required pybabel.cfg
 * Allow user to init a logger for missing translations and use `warning` as fallback
 * Fix or change build process in `bin/merge_catalogs` which looks for legacy gandi's catalogs
 * Replace pybabel by a npm script (eg. https://www.npmjs.com/package/babel-gettext-extractor)
-* Fix dependency path for scripts called in extract_messages (e.g. po2json)

--- a/package.json
+++ b/package.json
@@ -50,7 +50,6 @@
     "chai": "^3.5.0",
     "chai-enzyme": "^0.5.0",
     "cheerio": "^0.20.0",
-    "colors": "^1.1.2",
     "enzyme": "^2.4.1",
     "eslint": "^3.10.1",
     "eslint-config-airbnb": "^10.0.0",
@@ -61,10 +60,8 @@
     "eslint-plugin-react": "^6.7.1",
     "flow-bin": "^0.32.0",
     "jsdom": "^9.4.1",
-    "lodash": "^4.17.1",
     "ls-to-cp": "^2.1.0",
     "mocha": "^3.0.1",
-    "moment": "^2.15.1",
     "npm-run-all": "^3.1.1",
     "nyc": "^7.1.0",
     "react": "^15.3.1",
@@ -74,18 +71,19 @@
     "sinon": "^1.17.5"
   },
   "dependencies": {
+    "colors": "^1.1.2",
     "counterpart": "^0.17.2",
     "glob": "^7.0.5",
     "hoist-non-react-statics": "^1.2.0",
     "intl": "^1.2.5",
+    "lodash": "^4.17.1",
+    "moment": "^2.15.1",
     "po2json": "^0.4.1",
     "shallowequal": "^0.2.2",
     "warning": "^3.0.0"
   },
   "peerDependencies": {
-    "colors": "*",
-    "lodash": "*",
-    "moment": ">=2.15.1",
+    "babel-cli": "*",
     "react": "*",
     "react-dom": "*"
   }

